Analytics/Integrations/Flurry/SEGFlurryIntegration.m

```diff
@@ -62,6 +62,16 @@ - (void)identify:(NSString *)userId traits:(NSDictionary *)traits options:(NSDic
   if (age) {
     [Flurry setAge:[age intValue]];
   }
+  
+  NSDictionary *location = [traits objectForKey:@"location"];
+  
+  if (location) {
+    float latitude = [[location objectForKey:@"latitude"] floatValue];
```

Comment:
yep it would be 0. `nil` is magical like that in objc. if it where NSNull * on the other hand then it'd be a problem. this post here covers it in a bit more detail: http://nshipster.com/nil/